### DISCUSSION TITLE
Probing profile didn't save in probe tab

### DISCUSSION THREAD

**Discussion by rmaltais** [terjeio/ioSender#317] [General]
Hi,

When i created a new profile,it was no saving after an other session. I tried save in setting APP and save GRBL setting. I need to re-edit always my custom one. Maybe i do something wrong.

**Comment by rmaltais** [comment]
Sorry it's my mistake. I had install program as Administrator and run as user. Only a security issue in the XML file. I'd fix it.

**Comment by andymid3D** [comment]
Hi, 

I am seeing this issue, was using win10, now win11, app is running from user downloads folder directly... can update default settings, but not add a new one...? have tried running with as admin, even thou is in user space, still greyed out...
any ideas appreciated!

Andy
:-)

**Reply by terjeio** [reply]
Did you give the profile a name before clicking on the menu button?

**Reply by andymid3D** [reply]
I think I tried that and couldn't edit, but not sure.. on Wednesday I will try again, anything else that may be of note..? I do not have much time at the client installation, so would be good to go as prepared as possible! 
Also, I assume that the profile is that on the left of the probing tabs available, so is a all-covering profile that includes parameters that are universal for all types of probing.. I don't have to have had a successful probe completed before being able to save a profile name..? that would only apply to saving that probe / height map, etc..?

**Reply by andymid3D** [reply]
> Did you give the profile a name before clicking on the menu button?

Correct, this does let me create a new profile, and add it, and update this profile... 

Thanks again!
Andy
:-)

**Reply by terjeio** [reply]
> that would only apply to saving that probe / height map, etc..?

The profile data is only for the left hand side values of the main _Probing_ tab, values in the separate probing tabs are not saved. A height map can be saved/reloaded from the _Height map_ tab.
